### ISSUE TITLE
Issue: Token counts not returned by `list_runs`

### ISSUE THREAD

**Issue by shenghann** [langchain-ai/langsmith-sdk#225]
### Issue you'd like to raise.

Following the cookbook: https://github.com/langchain-ai/langsmith-cookbook/blob/main/exploratory-data-analysis/exporting-llm-runs-and-feedback/llm_run_etl.ipynb

`list_runs` is supposed to return `run.prompt_tokens`, `run.completion_tokens` and `run.total_tokens`. However all three is not present:

```python
runs = list(client.list_runs(
    project_name=project_name,
    run_type="llm",
    start_time=start_time,
))
```

Other info:
- hosted langsmith endpoint

```
'runtime': 'python',
  'platform': 'Linux-5.19.0-1025-aws-x86_64-with-glibc2.35',
  'sdk_version': '0.0.33',
  'library_version': '0.0.283',
  'runtime_version': '3.11.4',
  'langchain_version': '0.0.283',```


### Suggestion:

_No response_

**Comment by hinthornw** [comment]
Thanks for including the info! Could you confirm that the tokens also don't show up for runs logged > 10 minutes ago?

Some information is computed asynchronously so they may not be visible on runs logged in the past minute or so

**Comment by shenghann** [comment]
Yes, tokens missing even for runs > 10 minutes. Sorry I forgot to point that out!

**Comment by hinthornw** [comment]
Got it - and you're runs are logged using LangChain rather than the traceable decorator or another method? (I'm assuming yes given the env info)

If you view the run in the app, are tokens listed?

**Comment by shenghann** [comment]
Yes tokens are listed in the app - hence puzzling why they aren’t returned!

**Comment by hinthornw** [comment]
I haven't been able to reproduce this yet - do you feel comfortable sharing an example run to support@langchain.dev (via the public share link) that has these issues?

**Comment by hinthornw** [comment]
Hi @shenghann are you still seeing this issue on new runs? We've made some updates recently and wanted to check in

**Comment by shenghann** [comment]
Hey @hinthornw thanks for following up! Sorry I missed your earlier message to send some samples over.
I just pulled some runs and happy to report that the token counts are now returning. I did not update the my local Langsmith client - so I suppose it was the server side changes that fixed things.

Thank you!
